dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops

```diff
@@ -74,6 +74,10 @@ OPENTRACING_NO_TRACERFACTORY=CWMOT0008E: OpenTracing cannot track JAX-RS request
 OPENTRACING_NO_TRACERFACTORY.explanation=OpenTracing will not function properly because an OpentracingTracerFactory class was not provided. JAX-RS requests will execute normally, but they will not be tracked.
 OPENTRACING_NO_TRACERFACTORY.useraction=For more information, see the documentation about how to enable OpenTracing distributed tracing.
 
+OPENTRACING_TRACERFACTORY_NOT_PROVIDED=CWMOT0010W: OpenTracing cannot track JAX-RS requests because an OpentracingTracerFactory class was not provided.
+OPENTRACING_TRACERFACTORY_NOT_PROVIDED.explanation=JAX-RS requests will execute normally. But they will not be tracked because an OpentracingTracerFactory class was not provided.
+OPENTRACING_TRACERFACTORY_NOT_PROVIDED.useraction= This may be an acceptable behavior. For more information, see the documentation about how to enable OpenTracing distributed tracing.
```

Comment:
OPENTRACING_TRACERFACTORY_NOT_PROVIDED.useraction= See the documentation about how to enable and configure OpenTracing distributed tracing.  This warning can be safely ignored if OpenTracing is not being used.